dart/lib/src/protocol/sdk_info.dart

```diff
@@ -12,6 +14,33 @@ class SdkInfo {
     this.versionPatchlevel,
   });
 
+  /// Creates a SdkInfo out of a SdkVersion
+  factory SdkInfo.fromSdkVersion(SdkVersion sdkVersion) {
+    var name;
+    var major;
+    var minor;
+    var patch;
+    try {
+      name = sdkVersion.name;
+      final versions = sdkVersion.version.split('.');
+      final fullPatch = versions[2];
+      // because of prereleases (eg -alpha) sufix
+      final singlePatch = fullPatch.split('-');
+
+      major = int.parse(versions[0]);
+      minor = int.parse(versions[1]);
+      patch = int.parse(singlePatch[0]);
+    } catch (error) {
+      // something is wrong but keep going
+    }
+
+    return SdkInfo(
```

Comment:
This SDK info isnt' really out SDK. I'm not clear why symbolication needs it but it's about the builds tools and not the SDK itself (which is anyway in the event already)